doc/manuals/admin/database_admin.md

```diff
@@ -68,8 +68,7 @@ to take into account:
 
 Normally, Orion Context Broker uses just one database at MongoDB level
 (the one specified with the `-db` command line option, typically "orion").
-However, when [multitenant/multiservice is
-used](Publish/Subscribe_Broker_-_Orion_Context_Broker_-_User_and_Programmers_Guide#Multi_service_tenancy "wikilink")
+However, when [multitenant/multiservice](#multitenant/multiservice-database-separation) is used
```

Comment:
Commas are mandatory only if the sentence is trying to help understand the main sentence, which is "the behavior is different and... ", and is completely optional. See that without the first sentence the second would not make sense, hence the absence of a comma.